script.js

```diff
@@ -19,137 +19,290 @@ const state = {
   result: OPERATOR_EMPTY_STRING,
 };
 
-const buttons = document.querySelector(".buttons");
-const output = document.querySelector(".output");
+class Calculator {
+  constructor(container) {
+    this.container = container;
+    document.body.appendChild(this.container);
+  }
+
+  createCalculator() {
+    this.calc = document.createElement("div");
+
+    //this.calc.setAttribute("class", "calculator");
+
+    let value = document.getElementById("buttonAddCalculator").value;
+    this.calc.setAttribute("class", "calculator" + " " + value);
+
+    this.container.appendChild(this.calc);
 
-buttons.addEventListener("click", function (event) {
-  if (!event.target.classList.contains("number")) {
-    return;
+    document.getElementById("buttonAddCalculator").value++;
   }
 
-  state.checkValue = event.target.value;
-
-  if (
-    state.number1 === OPERATOR_EMPTY_STRING ||
-    state.number1.toString().slice(-1) === OPERATOR_POINT ||
-    state.operator === OPERATOR_EMPTY_STRING
-  ) {
-    state.number1 += state.checkValue;
-  } else if (
-    state.number1 !== OPERATOR_EMPTY_STRING &&
-    state.operator !== OPERATOR_EMPTY_STRING
-  ) {
-    state.number2 += state.checkValue;
+  createInput() {
+    this.input = document.createElement("input");
+
+    this.input.setAttribute("type", "text");
+    this.input.setAttribute("class", "output");
+    this.input.setAttribute("value", "");
+    this.input.setAttribute("disabled", "disabled");
+
+    this.calc.appendChild(this.input);
   }
 
-  output.value += state.checkValue;
-});
+  createButtons() {
+    this.buttons = document.createElement("div");
+
+    this.buttons.setAttribute("class", "buttons");
 
-buttons.addEventListener("click", function (event) {
-  if (!event.target.classList.contains("operator")) {
-    return;
+    this.calc.appendChild(this.buttons);
   }
 
-  state.checkValue = event.target.value;
+  createNumbers() {
+    this.numbers = document.createElement("div");
 
-  if (state.number1 === OPERATOR_EMPTY_STRING) {
-    state.checkValue = OPERATOR_EMPTY_STRING;
-  } else if (state.checkValue !== OPERATOR_EQUAL) {
-    if (
-      state.checkValue === OPERATOR_POINT ||
-      state.checkValue === OPERATOR_CLEAR
-    ) {
-      state.specialOperator = state.checkValue;
-    } else if (state.operator !== OPERATOR_EMPTY_STRING) {
-      state.operator = state.checkValue;
-      output.value = state.number1;
-    } else {
-      state.operator = state.checkValue;
+    this.numbers.setAttribute("class", "numbers");
+
+    this.buttons.appendChild(this.numbers);
+  }
+
+  createOperators() {
+    this.operators = document.createElement("div");
+
+    this.operators.setAttribute("class", "operators");
+
+    this.buttons.appendChild(this.operators);
+  }
+
+  createNumber() {
+    this.number = new Array(10);

```

Comment:
```suggestion
    this.numbers = new Array(10);

```
P.S. Уйдёт, если применишь массив, как я описывал